### PR TITLE
Python plugins

### PR DESCRIPTION
**Pull request by dnpwwo**
Add therostat support

### CODE REVIEW

**Comment by gizmocuz** [comment]
Thanks!

File: hardware/plugins/PluginMessages.h [hunk 1]
```diff
@@ -140,6 +142,21 @@ namespace Plugins {
 		DisconnectMessage(int HwdID) : CPluginMessage(PMT_Disconnect, HwdID) {};
 	};
 
+	class CommandMessage : public CPluginMessage
+	{
+	public:
+		CommandMessage(int HwdID, int Unit, const std::string & Message, const int level, const int hue) : CPluginMessage(PMT_Command, HwdID, Unit, Message, level, hue)
+		{
+			m_fLevel = -273.15;
```
Comment (oohlaf) [inline hardware/plugins/PluginMessages.h:150]: Ah, this is a bug! I was wondering why my onCommand method was always called with value -273.15! :)

Comment (dnpwwo) [reply hardware/plugins/PluginMessages.h:150]: Damn floating point inaccuracy!
-273.15 got stored as -273.149999999. So much for absolute zero.

Comment (oohlaf) [reply hardware/plugins/PluginMessages.h:150]: Ah, I see what you did, comparison to that value in both places
Here is a fix
https://github.com/domoticz/domoticz/pull/1245

**Pull request merged**